crates/node_binding/src/plugins/mod.rs

```diff
@@ -850,6 +856,50 @@ impl rspack_core::Plugin for JsHooksAdapter {
       .blocking_recv()
       .unwrap_or_else(|recv_err| panic!("{}", recv_err.to_string()))
   }
+
+  async fn runtime_module(
+    &self,
+    module: &mut dyn RuntimeModule,
+    chunk: &Chunk,
+    compilation: &Compilation,
+  ) -> rspack_error::Result<Option<String>> {
+    if self.is_hook_disabled(&Hook::RuntimeModule) {
+      return Ok(None);
+    }
+
+    self
+      .runtime_module_tsfn
+      .call(
+        JsRuntimeModuleArg {
+          module: JsRuntimeModule {
+            source: Some(
+              module
+                .generate(compilation)
+                .to_js_compat_source()
+                .expect("generate runtime module failed"),
+            ),
+            module_identifier: module.identifier().to_string(),
+            constructor_name: module.get_constructor_name(),
+            name: module
+              .identifier()
+              .to_string()
+              .replace("webpack/runtime/", ""),
+          },
+          chunk: JsChunk::from(chunk),
+        },
+        ThreadsafeFunctionCallMode::NonBlocking,
+      )
+      .into_rspack_result()?
+      .await
+      .unwrap_or_else(|err| panic!("Failed to call runtime module hook: {err}"))
+      .map(|r| {
+        r.and_then(|s| s.source).map(|s| {
+          std::str::from_utf8(&s.source)
+            .expect("Invalid utf-8 sequence")
```

Comment:
Should report this error (user's return value)

Reply:
Fixed by report the error with panic. But I think user's return value is a buffer and it is meanless to output it.